repo/db/db.go

```diff
@@ -16,25 +16,25 @@ var log = logging.MustGetLogger("db")
 
 type SQLiteDatastore struct {
 	config          repo.Config
-	followers       repo.Followers
-	following       repo.Following
-	offlineMessages repo.OfflineMessages
-	pointers        repo.Pointers
-	keys            wallet.Keys
-	stxos           wallet.Stxos
-	txns            wallet.Txns
-	utxos           wallet.Utxos
-	watchedScripts  wallet.WatchedScripts
-	settings        repo.Settings
-	inventory       repo.Inventory
-	purchases       repo.Purchases
-	sales           repo.Sales
-	cases           repo.Cases
-	chat            repo.Chat
-	notifications   repo.Notifications
-	coupons         repo.Coupons
-	txMetadata      repo.TxMetadata
-	moderatedStores repo.ModeratedStores
+	followers       repo.FollowerStore
+	following       repo.FollowingStore
+	offlineMessages repo.OfflineMessageStore
+	pointers        repo.PointerStore
+	keys            repo.KeyStore
+	stxos           repo.SpentTransactionOutputStore
+	txns            repo.TransactionStore
+	utxos           repo.UnspentTransactionOutputStore
+	watchedScripts  repo.WatchedScriptStore
+	settings        repo.ConfigurationStore
+	inventory       repo.InventoryStore
+	purchases       repo.PurchaseStore
+	sales           repo.SaleStore
+	cases           repo.CaseStore
+	chat            repo.ChatStore
+	notifications   repo.NotificationStore
+	coupons         repo.CouponStore
+	txMetadata      repo.TransactionMetadataStore
+	moderatedStores repo.ModeratedStore
```

Comment:
No more wallet interfaces to worry about here.
![](https://i.imgur.com/mMe5bkU.png)